src/main/java/com/epam/tc/hw/mobile/pageObjects/NativePageObject.java

```diff
@@ -4,61 +4,119 @@
 
     @AndroidFindBy(id = "platkovsky.alexey.epamtestapp:id/login_email")
-    WebElement loginEmail;
+    private WebElement andriodSignInEmail;
 
     @AndroidFindBy(id = "platkovsky.alexey.epamtestapp:id/login_pwd")
-    WebElement loginPwd;
+    private WebElement androidSignInPwd;
 
     @AndroidFindBy(id = "platkovsky.alexey.epamtestapp:id/registration_email")
-    WebElement registerEmail;
+    private WebElement androidRegEmail;
 
     @AndroidFindBy(id = "platkovsky.alexey.epamtestapp:id/registration_username")
-    WebElement registerUsername;
+    private WebElement androidRegName;
 
     @AndroidFindBy(id = "platkovsky.alexey.epamtestapp:id/registration_password")
-    WebElement registerPwd;
+    private WebElement androidRegPwd;
 
     @AndroidFindBy(id = "platkovsky.alexey.epamtestapp:id/registration_confirm_password")
-    WebElement registerConfirmPwd;
+    private WebElement androidRegConfirmPwd;
 
     @AndroidFindBy(id = "platkovsky.alexey.epamtestapp:id/register_new_account_button")
-    WebElement registerAccBtn;
+    private WebElement androidRegUserBtn;
 
     @AndroidFindBy(xpath = ".//android.view.ViewGroup/android.widget.TextView")
-    WebElement pageName;
+    private WebElement androidPageName;
+
+    @FindBy(xpath = ".//XCUIElementTypeSecureTextField")
+    private WebElement iosSignInPwd;
+
+    @FindBy(xpath = ".//XCUIElementTypeTextField")
+    private WebElement iosSignInEmail;
+
+    @FindBy(xpath = ".//*[@value= 'Sign In']")
+    private WebElement iosSignInBtn;
+
+    @FindBy(xpath = ".//*[@value= 'Register new account']")
+    private WebElement iosRegBtn;
+
+    @FindBy(xpath = ".//*[@value= 'user@example.com']")
+    private WebElement iosRegEmail;
+
+    @FindBy(xpath = ".//*[@value= 'TimApple']")
+    private WebElement iosRegName;
+
+    @FindBy(xpath = ".//*[@value= 'Required']")
+    private WebElement iosRegPwd;
+
+    @FindBy(xpath = ".//*[@value= 'Repeat please']")
+    private WebElement iosRegConfirmPwd;
+
+    @FindBy(xpath = ".//*[@type= 'XCUIElementTypeSwitch']")
+    private WebElement iosRegSwitch;
+
+    @FindBy(xpath = ".//*[@name= 'I read agreaments and agree wit it']")
+    private WebElement iosRegAgreament;
+
+    @FindBy(xpath = ".//*[@value= 'Register new account']")
+    private WebElement iosRegUserBtn;
+
+    @FindBy(xpath = ".//XCUIElementTypeNavigationBar/XCUIElementTypeStaticText")
+    private WebElement iosPageName;
 
     public NativePageObject(AppiumDriver<?> appiumDriver) {
         PageFactory.initElements( new AppiumFieldDecorator(appiumDriver), this);
     }
 
-    public void registerUser(String email, String name, String password) {
-        registerBtn.click();
-        registerEmail.sendKeys(email);
-        registerUsername.sendKeys(name);
-        registerPwd.sendKeys(password);
-        registerConfirmPwd.sendKeys(password);
-        registerAccBtn.click();
+    public void registerIosUser(String email, String name, String password) {
```

Comment:
It's better to group email, name and password as a business object - separate class that can be manipulated like an entity